equinox/nn/normalization.py

```diff
@@ -0,0 +1,69 @@
+from typing import List, Optional, Tuple, Union
+
+import jax
+import jax.numpy as jnp
+
+from ..custom_types import Array
+from ..module import Module, static_field
+
+
+_shape_t = Union[int, Tuple[int], List[int]]
+
+
+class LayerNorm(Module):
+    """Layer Normalization as described in https://arxiv.org/abs/1607.06450"""
```

Comment:
I think it'd be great to have the precise mathematical description here, c.f. [here](https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html#torch.nn.LayerNorm), including the precise meaning of `elementwise_affine`. (By all means just copy what they've written verbatim if you do/don't wish.)

Reply:
(And the meaning of `normalized_shape`, actually.)